apps/discord-bot/src/models/bot.ts

```diff
@@ -90,6 +95,20 @@ export class Bot {
 
     this.ready = true
     Logger.info(Logs.info.clientReady)
+
+    const ctaChannel = this.client.guilds.cache.find(dggPol => dggPol.name === guildName)?.channels.cache.find(ctaChan => ctaChan?.name === ctaChannelName);
+
+    if (ctaChannel?.type === ChannelType.GuildText) {
+      // eslint-disable-next-line no-console
+      console.log("bot started... checking for cta message...");
+      const ctaPostTrigger = new CTAPostTrigger();
+
+      ctaChannel.messages.fetch().then(msgs => {
```

Comment:
Using `.then` here works, but it's already an async function, so we can also use `await` too which is a bit easier to read.